bundles/org.openhab.binding.insteon/src/main/resources/device_types.xml

```diff
@@ -76,6 +76,8 @@ Example entry:
      <model>2842-222</model>
      <description>Motion Sensor</description>
      <feature name="contact">WirelessMotionSensorContact</feature>
+     <feature name="dawndusk">WirelessMotionSensorDawndusk</feature>
+     <feature name="lowbat">WirelessMotionSensorLowbat</feature>
```

Comment:
Can we use something more generic than "dawndusk"? This assumes that the variable is associated with the sun rise/set.
lowbat->lowbattery

Reply:
I can make it whatever you'd like, but I think this is the traditional way to refer to this feature on motion sensors going back to old X10 stuff. How about lightdark?

Lowbat - done

Reply:
changed to <feature name="lightdark">WirelessMotionSensorLightdark</feature>